assets/fonts/OpenSans/font-face.css

```diff
@@ -0,0 +1,11 @@
+@font-face {
```

Comment:
I'm not too familiar with CSS, so correct me if I'm wrong: Can't we use the font-weight property to make the other font lighter?